### PR TITLE
Release 0.9.1

### PR DESCRIPTION
**Pull request by ajnelson-nist**
This Pull Request resolves part of Issue #437 .

# Coordination

- [x] Downstream CI review finished on [PR 484](https://github.com/ucoProject/UCO/pull/484)

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
@eoghanscasey and/or @sbarnum : Your review would be appreciated on this PR.  At least one of you will need to log an approving review before I can merge it, per 2-key rule.  This review is non-trivial, so I appreciate there might be some discussion.

The sibling review to merge back into `develop` has a bit more complexity layered onto it that needs to come after this PR is merged, so your review of this PR does not necessarily cover [PR 481](https://github.com/ucoProject/UCO/pull/481).  I'm still working on last polish on that PR.

**Comment by ajnelson-nist** [comment]
Please do not merge yet, I am doing some further testing with CASE consuming this practice.

**Review by eoghanscasey — APPROVED** [review]
This versioning follows general practice for ontology versioning, which is desirable. Implementers can choose whether to represent all data in the current version (converting prior versions to current), or to take on the added overhead of using versioning to combine data from different compatible versions.

**Pull request merged**
